packages/test/test-drivers/src/routerliciousTestDriver.ts

```diff
@@ -51,10 +64,11 @@ function getConfig(
 }
 
 function getLegacyConfigFromEnv() {
+    const discoveryEndpoint = process.env.fluid__webpack__discoveryEndpoint;
     const fluidHost = process.env.fluid__webpack__fluidHost;
     const tenantSecret = process.env.fluid__webpack__tenantSecret;
     const tenantId = process.env.fluid__webpack__tenantId ?? "fluid";
-    return getConfig(fluidHost, tenantId, tenantSecret);
+    return getConfig(discoveryEndpoint, fluidHost, tenantId, tenantSecret);
 }
 
 function getEndpointConfigFromEnv(r11sEndpointName: RouterliciousEndpoint) {
```

Comment:
if `r11sEndpointName === "docker"`, should it also use the discoveryEndpoint? If so, `dockerConfig()` should be updated as well

Reply:
When the r11sEndpointName is docker, it won't use the discovery endpoint, since all that happen is in the local docker which won't use the discovery feature to use multiple servers.

Reply:
It still works though, right? It just always returns a config value for the endpoints?

Reply:
Yes, it should still work, and return the correct config values.